### PR TITLE
Update field descriptions of DaemonSet rolling udpate

### PR DESCRIPTION
**Pull request by janetkuo**
@kargakis @lukaszo @kubernetes/sig-apps-bugs

### CODE REVIEW

**Comment by janetkuo** [comment]
Fix verification failures by auto-generate files. No other changes

**Comment by janetkuo** [comment]
Still auto generate only

**Comment by janetkuo** [comment]
Rebased

**Comment by kargakis** [comment]
@janetkuo needs one more rebase

File: pkg/apis/extensions/v1beta1/types.go [hunk 1]
```diff
@@ -403,13 +403,14 @@ type RollingUpdateDaemonSet struct {
...
-	// Example: when this is set to 30%, 30% of the currently running DaemonSet
-	// pods can be stopped for an update at any given time. The update starts
-	// by stopping at most 30% of the currently running DaemonSet pods and then
-	// brings up new DaemonSet pods in their place. Once the new pods are ready,
-	// it then proceeds onto other DaemonSet pods, thus ensuring that at least
-	// 70% of original number of DaemonSet pods are available at all times
-	// during the update.
+	// Example: when this is set to 30%, at most 30% of the total number of nodes
+	// that should be running the daemon pod (i.e. DesiredNumberScheduled in
+	// DaemonSetStatus) can have their pods stopped for an update at any given
```
Comment (bgrant0607) [inline pkg/apis/extensions/v1beta1/types.go]: Please use json field names everywhere in types.go comments.

**Comment by janetkuo** [comment]
PTAL, the 2nd commit changes the descriptions of DaemonSet related go field names to json field names.

**Comment by bgrant0607** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**